<launch>
  <param name="use_sim_time" value="false" />

  <node	pkg="sdf_slam" type="sdf_slam_node" name="sdf_2d_node" output="screen">
    <param name="p_timeout_ticks_" value="0" />	    
    <param name="p_update_map_aoe_" value="5" />	    
    
    <param name="p_initial_pose_" type="bool" value="false" />	    
    <param name="p_publish_map_" type="bool" value="true" />	    

    <param name="p_avg_mapping_" type="bool" value="true" />	    
    <param name="p_fine_pos_" type="bool" value="true" />	    
    <param name="p_perma_map_" type="bool" value="false" />	    

    

    <param name="p_robot_frame_" type="string" value="Robot_1/base_link" />	    
    <param name="p_fixed_frame_" type="string" value="world" />	    
    <param name="p_tar_frame_" type="string" value="sdf_pos" />	    


    <param name="p_min_range_" value="0.4" />	    


  </node>

</launch>
